packages/itwin/tree-widget/package.json

```diff
@@ -100,12 +104,12 @@
     "xmlhttprequest": "^1.8.0"
   },
   "peerDependencies": {
-    "@itwin/appui-abstract": "^3.6.0",
-    "@itwin/appui-react": "^3.6.0",
-    "@itwin/components-react": "^3.6.0",
-    "@itwin/core-frontend": "^3.6.0",
-    "@itwin/core-react": "^3.6.0",
-    "@itwin/presentation-components": "^3.6.0",
+    "@itwin/appui-abstract": "^3.7.0",
```

Comment:
For the most immediate `tree-widget-react` dev release, let's have these peer dependencies at the latest available dev versions. Afterwards, we'll bump them to regular versions and make another, non-dev, release.